app/src/main/res/layout/fragment_subscription_details.xml

```diff
@@ -59,23 +26,24 @@
 
       <ImageView
           android:id="@+id/app_icon"
-          android:layout_width="96dp"
-          android:layout_height="96dp"
+          android:layout_width="64dp"
```

Comment:
If many components have the same margins due to the way the design is made we should try to make sure then when of them is changed the others are also changed. This it's easier to achieve when using a value in the dimens file